src/express/commands/auth-bank-tests.js

```diff
@@ -0,0 +1,254 @@
+    `${doc.ethHostUser}@${machine0}`,
+    `echo '${metadataJson}' > ~/draft_metadata.json`,
+    maxRetries
+  )
+  console.log(`✅ draft_metadata.json saved on host ${machine0}`)
+
+  console.log('📍Writing draft_proposal.json on primary host:', machine0)
+  await runSshCommand(
+    `${doc.ethHostUser}@${machine0}`,
+    `echo '${proposalJson}' > ~/draft_proposal.json`,
+    maxRetries
+  )
+  console.log(`✅ draft_proposal.json saved on host ${machine0}`)
+
+  const chainId = await runSshCommandWithReturn(
+    `${doc.ethHostUser}@${machine0}`,
+    "jq -r '.chain_id' /var/lib/heimdall/config/genesis.json",
+    maxRetries
+  )
+  console.log('Chain ID:', chainId.trim())
+
+  // Check proposal count before submission
+  const beforeCount = await getProposalCount(doc, machine0)
+  console.log('🔍 Proposals before submission:', beforeCount)
+
+  const submitProposalCommand = `printf 'test-test\\n' | heimdalld tx gov submit-proposal draft_proposal.json --from test --home /var/lib/heimdall/ --chain-id ${chainId.trim()} -y`
+  await runSshCommand(
+    `${doc.ethHostUser}@${machine0}`,
+    submitProposalCommand,
+    maxRetries
+  )
+
+  await timer(2000)
+
+  // Check proposal count after submission
+  const afterCount = await getProposalCount(doc, machine0)
+  console.log('🔍 Proposals after submission:', afterCount)
+
+  if (afterCount > beforeCount) {
+    console.log('✅ Proposal submitted successfully')
+  } else {
+    console.log('❌ Proposal submission failed')
+  }
+
+  console.log(`📍Depositing 200 POL to proposal #${afterCount}`)
+  const depositCommand = `printf 'test-test\\n' | heimdalld tx gov deposit ${afterCount} 200000000000000000000pol --from test --home /var/lib/heimdall/ --chain-id ${chainId.trim()} -y`
+  for (const machine of doc.devnetBorHosts) {
+    await runSshCommand(
+      `${doc.ethHostUser}@${machine}`,
+      depositCommand,
+      maxRetries
+    )
+    console.log(`✅ Deposit command executed on host ${machine}`)
+    await timer(2000)
+  }
+
+  console.log(
+    `📍 Casting YES vote on proposal #${afterCount} from each validator…`
+  )
+  for (const machine of doc.devnetBorHosts) {
+    const voteCommand = `printf 'test-test\\n' | heimdalld tx gov vote ${afterCount} yes --from test --home /var/lib/heimdall/ --chain-id ${chainId.trim()} -y`
+    runSshCommand(`${doc.ethHostUser}@${machine}`, voteCommand, maxRetries)
+    console.log(`✅ Vote command executed on host ${machine}`)
+    await timer(2000)
+  }
+
+  await timer(75000) // Wait for 75 secs
+  console.log('📍Checking proposal status…')
+
+  // Check proposal status
+  const status = await getProposalStatus(doc, machine0, afterCount)
+  if (status === 'PROPOSAL_STATUS_PASSED') {
+    console.log('✅ Proposal passed successfully')
+  } else {
+    console.error(`❌ Proposal status: ${status}`)
+  }
+
+  console.log('✅ auth.MsgUpdateParam Testcase passed')
+
+  console.log('📍 Verifying auth params updated')
+  const ok = await verifyAuthParamsUpdate(doc, machine0)
```

Comment:
Can we fetch the params from remote before the proposal is submitted and verify they differ from these ones? (and they match afterwards as you are already doing)